src/main/java/net/neoforged/neoforge/common/world/BiomeSpecialEffectsBuilder.java

```diff
@@ -30,6 +30,7 @@ public static BiomeSpecialEffectsBuilder copyOf(BiomeSpecialEffects baseEffects)
         baseEffects.getAmbientMoodSettings().ifPresent(builder::ambientMoodSound);
         baseEffects.getAmbientAdditionsSettings().ifPresent(builder::ambientAdditionsSound);
         baseEffects.getBackgroundMusic().ifPresent(builder::backgroundMusic);
+        builder.backgroundMusicVolume = baseEffects.getBackgroundMusicVolume();
```

Comment:
I would avoid setting the field directly in case vanilla adds side effects later.

```suggestion
        builder.backgroundMusicVolume(baseEffects.getBackgroundMusicVolume());
```

Reply:
Well, the PR is consistent with how `grassColorModifier` is set above.

Reply:
Oh, true.